pkg/motor/login.go

```diff
@@ -47,13 +47,13 @@ func (mtr *motorNodeImpl) LoginWithKeys(request mt.LoginWithKeysRequest) (mt.Log
 	mtr.Address = request.Did
 
 	// fetch vault shards
-	mtr.callback.OnMotorEvent("Fetching shards from vault", false)
+	// TODO: Breaking (bind.ios) mtr.callback.OnMotorEvent("Fetching shards from vault", false)
```

Comment:
could there be a check on the callback instance for the `OnMotorEvent` being nil instead of commenting out?